services/core-web/src/constants/routes.ts

```diff
@@ -355,6 +355,13 @@ export const REPORT_VIEW_EDIT = {
   component: ReportPage,
 };
 
+export const MINE_REPORT_VIEW_EDIT = {
```

Comment:
Is there a reason not to use the REPORT_VIEW_EDIT route above?

Reply:
oh reverted that. Can use the REPORT_VIEW_EDIT for it.